linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml

```diff
@@ -0,0 +1,50 @@
+documentation_complete: true
+
+prodtype: fedora
+
+title: 'Ensure Fedora GPG Key Installed'
+
+description: |-
+    To ensure the system can cryptographically verify base software
+    packages come from Fedora (and to connect to the Fedora Network to
+    receive them), the Fedora GPG key must properly be installed.
+    To install the Fedora GPG key, run one of the commands below, depending on your Fedora vesion:
+    <pre>$ sudo rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-{{{ latest_version }}}-primary</pre>"
+    <pre>$ sudo rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-{{{ previous_version }}}-primary</pre>"
+
+rationale: |-
+    Changes to software components can have significant effects on the
+    overall security of the operating system. This requirement ensures
+    the software has not been tampered with and that it has been provided
+    by a trusted vendor. The Fedora GPG key is necessary to
+    cryptographically verify packages are from Fedora."
+
+severity: high
+
+identifiers:
+    cce@rhel6: 26506-6
+    cce@rhel7: 26957-1
+
+references:
+    stigid@rhel6: RHEL-06-000008
+    srg@rhel6: SRG-OS-000090
+    disa@rhel6: '351'
+    cis: 1.2.3
+    cjis: 5.10.4.1
+    cui: 3.4.8
+    disa: "1749"
+    hipaa: 164.308(a)(1)(ii)(D),164.312(b),164.312(c)(1),164.312(c)(2),164.312(e)(2)(i)
+    nist: CM-5(3),SI-7,MA-1(b)
+    ospp@rhel7: FAU_GEN.1.1.c
+    pcidss: Req-6.2
+    srg: "366"
```

Comment:
This srg doesn't look right to me.

Reply:
@redhatrises I don't know what to put there, nor where to get the right value, so I'll be removing the `srg` field.